pkg/reconciler/common/transformers.go

```diff
@@ -58,6 +59,11 @@ const (
 
 	runAsNonRootValue              = true
 	allowPrivilegedEscalationValue = false
+
+	tektonPipelinesControllerName                      = "tekton-pipelines-controller"
+	tektonPipelinesServiceName                         = "tekton-pipelines-controller"
+	tektonPipelinesControllerStatefulServiceName       = "STATEFUL_SERVICE_NAME"
+	tektonPipelinesControllerStatefulControllerOrdinal = "STATEFUL_CONTROLLER_ORDINAL"
```

Comment:
Pipelines specific things we can keep it in pipelines reconciler and the function can take these values as parameters so that we can use it for other components in future

Reply:
well, these constant are use in the transformers function in this file, so i think we should keep them here

Reply:
Yeah, that make sense, but let's say in future we want to add for other components as well, then we might not be able to reuse the transformer function